.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: ruby
 rvm:
-  - "2.0.0"
```

Comment:
ok, I think it works, but I think circleci has issues because the default bundler from that old needs updating.

FYI.. 2.0.0 isn't even getting security patches at this point.